### DISCUSSION TITLE
Code Review IM-11

### DISCUSSION THREAD

**Discussion by danil2205** [HowProgrammingWorks/LiveQA#54] [Code review]
Danil Shostak, IM-11
Пишу браузерную игру: "Space Invaders".
Суть игры: облетать объекты, собирать айтемы (космонавтов) и убивать боссов. По сути, игра бесконечная. Чем больше игрок живет и убивает боссов, тем больше он получает монеток, которые он может потратить в магазине на жизни и улучшения способностей. Поэтому игрок всегда может улучшить свой предыдущий результат. Также сохраняется его прогресс, тому после каждого перезахода в игру не придется начинать все сначала.
Описание игры пока не делал. Буду благодарен за ревью кода [тык](https://github.com/danil2205/Space-Invaders)

**Comment by tshemsedinov** [comment]
https://github.com/danil2205/Space-Invaders/issues/1

**Reply by danil2205** [reply]
All fixed.
